content/docs/0.13.x/operate/user_management/microsoft/index.md

```diff
@@ -33,15 +35,28 @@ OAUTH_CLIENT_SECRET: <client_secret>
 OAUTH_SCOPE: "email"
 ```
 
-* When accessing the Bridge, the user is redirected to the identity provider.
+When accessing the Bridge, the user is redirected to the identity provider.
 {{< popup_image
 link="./assets/oauth-login-message.png"
 caption="Accessing bridge without being logged in" width="800px">}}
 {{< popup_image
 link="./assets/oauth-login.png"
 caption="Entering user credentials" width="400px">}}
 
-* After the user successfully logs in with his Microsoft credentials, the user is redirected back to the Bridge. Once redirected, the Bridge server fetches the user tokens and creates a session. The user is now successfully logged in.
+After the user successfully logs in with his Microsoft credentials, the user is redirected back to the Bridge. Once redirected, the Bridge server fetches the user tokens and creates a session. The user is now successfully logged in.
 {{< popup_image
 link="./assets/oauth-logged-in.png"
 caption="User is logged in" width="800px">}}
+
+## Keptn CLI
+
+The `keptn auth` command provides several [command line flags](../../../reference/cli/commands/keptn_auth) which can be used to setup the CLI to use SSO via OpenID:
```

Comment:
this link does not work

Reply:
same as above, why do you think it's not working?